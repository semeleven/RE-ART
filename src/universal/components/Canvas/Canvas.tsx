import styled from '@Styled';

const StyledCanvas = styled.div`
	position: relative;
	min-height: 100vh;
`;

export default StyledCanvas;
