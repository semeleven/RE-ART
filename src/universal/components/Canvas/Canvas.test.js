import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Canvas from './Canvas';

it('renders Canvas component', () => {
	const wrapper = shallow(
		<Canvas>
			Content
		</Canvas>
	);
	expect(wrapper).toMatchSnapshot();
});
