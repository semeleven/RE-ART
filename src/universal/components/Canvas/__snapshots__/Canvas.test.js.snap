// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Canvas component 1`] = `
.c0 {
  position: relative;
  min-height: 100vh;
}

<div
  className="c0"
>
  Content
</div>
`;
